metrics/internal/collectors/ceph-block-pool.go

```diff
@@ -108,9 +110,15 @@ func (c *CephBlockPoolCollector) collectMirroringImageHealth(cephBlockPools []*c
 	for _, cephBlockPool := range cephBlockPools {
 		var imageHealth string
 		mirroringStatus := cephBlockPool.Status.MirroringStatus
-		if mirroringStatus != nil {
-			imageHealth = mirroringStatus.Summary.ImageHealth
+		if mirroringStatus == nil || mirroringStatus.Summary == nil {
```

Comment:
@umangachapagain can we check https://github.com/rook/rook/blob/df04e5cce53f7b4c34cc060877b09cae2a5e4ffc/deploy/examples/pool.yaml#L49 her as well as the mirroring need to enabled on the pool to check anything?

Reply:
Good catch. Done.